editor/src/clj/editor/welcome.clj

```diff
@@ -91,12 +91,25 @@
 (def ^:private WelcomeSettings
   {:new-project NewProjectSettings})
 
-(defn- load-edn [path]
-  (with-open [reader (PushbackReader. (io/reader (io/resource path)))]
+(defn- load-edn [edn]
```

Comment:
What _is_ edn?
Is it a path?
If it's a path, then I think it was named correctly.
As for "edn" I don't know how to interpret that.
I know clojure isn't great at conveying the type unless we specify it explicitly.

If it's a resource, then I think "resource" is good, as that's a common type in the editor, and you'll get the familiarity that goes with it.
Also note that the function is already called "load-edn" so the "edn" part is already explicit.

The further down I read, I eventually notice it might be a FIle ?
Perhaps put the type here as well?

Reply:
It can actually be almost "anything". Docs for io/reader:

Attempts to coerce its argument into an open java.io.Reader. Default implementations always return a java.io.BufferedReader. Default implementations are provided for Reader, BufferedReader, InputStream, File, URI, URL, Socket, byte arrays, character arrays, and String.
If argument is a String, it tries to resolve it first as a URI, then as a local file name.  URIs with a 'file' protocol are converted to local file names.

I'll name it `edn-resource`.

Reply:
We use load-edn both to load from a java.io.File (ie from user home folder) and as a Resource (from inside the jar).